src/poliastro/twobody/orbit.py

```diff
@@ -238,14 +238,18 @@ def from_body_ephem(cls, body, epoch=None):
         return ss
 
     @classmethod
-    def from_horizons(cls, name, epoch=None, plane=Planes.EARTH_EQUATOR):
+    def from_horizons(
+        cls, name, epoch=None, plane=Planes.EARTH_EQUATOR, id_type="smallbody"
```

Comment:
I don't like to expose this to the user, but the notebook doesn't work otherwise. A necessary evil, I guess!

Reply:
Not just notebook, user won't be able to get the orbit of `major body` if the parameter is not set.